src/commands/pack/macos.ts

```diff
@@ -1,21 +1,20 @@
-import * as path from 'path'
-
-import * as _ from 'lodash'
-import * as fs from 'fs-extra'
 import {Command, Flags, Interfaces} from '@oclif/core'
+import * as fs from 'fs-extra'
+import {exec as execSync} from 'node:child_process'
+import * as os from 'node:os'
+import * as path from 'node:path'
+import {promisify} from 'node:util'
 
 import * as Tarballs from '../../tarballs'
 import {templateShortKey} from '../../upload-util'
-import {exec as execSync} from 'child_process'
-import {promisify} from 'node:util'
-import * as os from 'os'
+import {uniq} from '../../util'
 
 const exec = promisify(execSync)
 type OclifConfig = {
```

Comment:
here, and above where you create types for jitPlugins:

Would it be better to update the pjson/oclif typing in `oclif/core` so that `oclif/oclif` can reference those?